test/lib/send_with_us/api_request_test.rb

```diff
@@ -68,14 +53,29 @@ def test_send_with_connection_refused
     assert_raises( SendWithUs::ApiConnectionRefused ) { @request.post(:send, @payload) }
   end
 
-  def test_send_with_version
+  def test_send_with_with_attachment
+    build_objects
+    email_id = 'test_fixture_1'
+    result = @api.send_with(
+      email_id,
+      {name: 'Ruby Unit Test', address: 'matt@example.com'},
+      {data: 'I AM DATA'},
+      {name: 'sendwithus', address: 'matt@example.com'},
+      [],
+      [],
+      ['README.md']
+    )
+    assert_instance_of( Net::HTTPOK, result )
```

Comment:
As the client(s) evolve we'll certainly move towards proper test suites.  Thanks for the suggestions and tips.